<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="use_cpu"           value="true" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov8_ros" type="yolo_v8_right.py" name="yolov8_ros_right" output="screen" >
    <param name="weight_path"       value="$(find yolov8_ros)/weights/last.pt"/>
    <!-- 待修改 -->
    <!-- <param name="left_image_topic"       value="/left_cam/image_raw" /> -->
    <param name="right_image_topic"       value="/right_cam/image_raw" />
    <!-- 待修改 -->
    <!-- <param name="left_pub_topic"         value="/left/yolov8/BoundingBoxes" /> -->
    <param name="right_pub_topic"         value="/right/yolov8/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="visualize"       value="true"/>
    <param name="conf"              value="0.3" />
  </node>

</launch>
